src/mongocxx/collection.cpp

```diff
@@ -65,6 +65,20 @@ stdx::string_view collection::name() const noexcept {
     return _impl->name;
 }
 
+void collection::rename(stdx::string_view new_name, bool drop_target_before_rename) {
+  bson_error_t error;
+
+  auto result = mongoc_collection_rename(_impl->collection_t, _impl->database_name.c_str(),
```

Comment:
Hi @koraa , please also add this symbol to the list in `src/mongocxx/private/libmongoc_symbols.hpp` and then call `libmongoc::collection_rename` instead of calling `mongoc_collection_rename` directly.